xamarin/FHXamarinIOSSDK/FHClient.cs

```diff
@@ -60,6 +61,18 @@ public class FHClient: FH
 			return await FH.Init ();
 		}
 
+		public static void FinishRegistration(NSData deviceToken)
+		{
+			var notification = NSNotification.FromName("sucess_registered", deviceToken);
+			NSNotificationCenter.DefaultCenter.PostNotification (notification);
```

Comment:
@secondsun that way we don't have to couple the FHClient class to the Push class and it can wait for the deviceToken to finish the registration